src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { addressBookImport } from 'src/logic/addressBook/store/actions'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { MigrationMessage } from 'src/routes/migration/container'
+import { MIGRATION_ADDRESS } from 'src/routes/routes'
+
+const MAINET_URL = 'http://localhost:3000'
+const networks = [
+  {
+    safeUrl: 'http://localhost:3001/#',
+  },
+  {
+    safeUrl: 'http://localhost:3002/#',
```

Comment:
Need to update these URLs.

Reply:
After QA we are going to read the urls from the networks provided by getNetworks() function